package.json

```diff
@@ -13,8 +13,8 @@
   },
   "private": true,
   "scripts": {
-    "lint": "eslint '**/*.js'",
```

Comment:
Can we also remove the direct dependencies on `eslint`, `prettier`, and any other `eslint` plugins we're using and now don't need?

Reply:
good catch, thanks.